common/src/main/java/dev/latvian/mods/kubejs/item/ItemBuilder.java

```diff
@@ -131,6 +134,13 @@ public void clientRegistry(Minecraft minecraft) {
 		if (!color.isEmpty()) {
 			ColorHandlerRegistry.registerItemColors((stack, index) -> color.get(index), this);
 		}
+
+		if(colorCallback != null) {
+			ColorHandlerRegistry.registerItemColors((itemStack, tintIndex) -> {
+				Object color = colorCallback.getColor(new ItemStackJS(itemStack), tintIndex);
+				return ColorWrapper.of(color).getRgbKJS();
+			}, this);
+		}
```

Comment:
Could we somehow merge these two into the same property? I don't really like the thought of maintaining two completely separate ways of assigning colours to an item

Reply:
Makes totally sense to merge them